xonsh/built_ins.py

```diff
@@ -569,9 +569,9 @@ def run_subproc(cmds, captured=False):
                     e = 'xonsh: subprocess mode: permission denied: {0}'
                     raise XonshError(e.format(cmd[0]))
         _stdin_file = None
-        if (captured == 'object' and
-                stdin is not None and
-                __xonsh_env__['XONSH_STORE_STDIN'] and
+        if (stdin is not None and
+                ENV.get('XONSH_STORE_STDIN') and
```

Comment:
It is just a consistency thing. On the `andor` branch I have cleaned up a lot of these thing.  If we wanted to move to indexing everywhere in the code base, that could also be OK